internal/pipe/cask/templates/cask.rb

```diff
@@ -0,0 +1,100 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+{{ if .CustomRequire -}}
+require_relative "{{ .CustomRequire }}"
+{{ end -}}
+cask "{{ .Name }}" do
+  desc "{{ .Desc }}"
+  homepage "{{ .Homepage }}"
+  version "{{ .Version }}"
+  {{- with .License }}
+  license "{{ .}}"
+  {{- end }}
+
+  {{- with .Binary }}
+  binary "{{ .}}"
+  {{- end }}
+  {{- with .Manpage }}
+  manpage "{{ .}}"
+  {{- end }}
+  {{- if .BashCompletions }}
+  bash_completions "{{ .BashCompletions }}"
+  {{- end }}
+  {{- if .ZshCompletions }}
+  zsh_completions "{{ .ZshCompletions }}"
+  {{- end }}
+  {{- if .FishCompletions }}
+  fish_completions "{{ .FishCompletions }}"
+  {{- end }}
+
+  {{- with .Dependencies }}
+  {{ range $index, $element := . }}
+  depends_on "{{ .Name }}"
+  {{- if .Type }} => :{{ .Type }}{{- else if .Version }} => "{{ .Version }}"{{- end }}
+  {{- with .OS }} if OS.{{ . }}?{{- end }}
+  {{- end }}
+  {{- end -}}
+
+  {{- if and .MacOSPackages .LinuxPackages }}
+  on_macos do
+  {{- include "macos_packages" . | indent 2 }}
+  end
+
+  on_linux do
+  {{- include "linux_packages" . | indent 2 }}
+  end
+  {{- end }}
+
+  {{- if and (.MacOSPackages) (not .LinuxPackages) }}
+  {{- template "macos_packages" . }}
+  {{- end }}
+
+  {{- if and (not .MacOSPackages) (.LinuxPackages) }}
+  {{- template "linux_packages" . }}
+  {{- end }}
+
+  {{- with .Conflicts }}
+
+  {{ range $index, $element := . }}
+  conflicts_with "{{ . }}"
+  {{- end }}
+  {{- end }}
+
+  {{- with .CustomBlock }}
+
+  {{ range $index, $element := . }}
+  {{ . }}
+  {{- end }}
+  {{- end }}
+
+  {{- with .PostInstall }}
+
+  def post_install
+    {{- range . }}
+    {{ . }}
+    {{- end }}
+  end
+  {{- end -}}
+
+  {{- with .Caveats }}
+
+  def caveats
+    <<~EOS
+    {{- range $index, $element := . }}
+      {{ . -}}
+    {{- end }}
+    EOS
+  end
+  {{- end -}}
+
+  {{- with .Service }}
+
+  service do
```

Comment:
Service blocks are not supported, but there is a service stanza that takes a file